.env

```diff
@@ -0,0 +1,8 @@
+REACT_APP_API_KEY='AIzaSyDs-rxvOLmskYhudQWPOxl-whELlKKUYhs',
+REACT_APP_AUTHDOMAIN="issue-finder.firebaseapp.com",
+REACT_APP_PROJECTID='issue-finder',
+REACT_APP_STORAGEBUCKET='issue-finder.appspot.com',
+REACT_APP_MSG_SENDERID='918268898722',
+REACT_APP_APPID='1:918268898722:web:653b04454a9eee41d896fb',
+REACT_APP_MEASUREMENTID='G-C28740E7D5'
+ 
```

Comment:
Great you have appropriate changes but don't commit the .env file to the repository rather put everything in the .env file in the .env.sample.

Reply:
you are doing great. Delete this .env file and commit. As I have mentioned earlier we don't want the .env file in the repository.